sky/cli.py

```diff
@@ -1876,6 +1875,51 @@ def down(
                            purge=purge)
 
 
+def _hints_for_down_spot_controller(controller_name: str):
+    # spot_jobs will be empty when the spot cluster is not running.
+    cluster_status, _ = backend_utils.refresh_cluster_status_handle(
+        controller_name)
+    if cluster_status is None:
+        click.echo('Managed spot controller has already been torn down.')
+        return
+
+    msg = (f'{colorama.Fore.YELLOW}WARNING: Tearing down the managed '
+           f'spot controller ({cluster_status.value}). Please be '
+           f'aware of the following:{colorama.Style.RESET_ALL}'
+           '\n * All logs and status information of the spot '
+           'jobs will be lost.')
```

Comment:
nit: ...jobs (output of `sky spot status`) will be lost.

Reply:
Done. Thanks!